niceman/retrace/packagemanagers.py

```diff
@@ -247,6 +250,12 @@ def _get_packages_for_files(self, files):
                 out = out.decode()
             except AttributeError:
                 pass
+            except UnicodeDecodeError as exc:
```

Comment:
What interesting case led to this?  (This is a reminder for me to ask tomorrow...)